docs/Cloud_integrations/LoRaWAN/LoRagateway.md

```diff
@@ -0,0 +1,101 @@
+
+    !!! Info
+        Connecting devices via the Bootstrap Server is unavailable in the Developer plan.
+
+4. Follow the steps in the device creator.
+
+Adding a LoRaWAN Gateway via the Management Server consists of two steps: 
+
+- **Step 1: Configure your device connection**
+- **Step 2: Connect your device** 
+
+Adding a LoRaWAN Gateway via the Bootstrap Server consists of three steps: 
+
+- **Step 1: Configure your device connection** 
+- **Step 2: Configure server connection**
+- **Step 3: Connect your device**
+
+Below you can find the description of each step:
+
+### Configure your device connection
+		
+In this step provide the information about the LoRaWAN Gateway device and choose device connection settings.
+		
+In the section *Device credentials*, provide the following information:
+
+- *Gateway EUI* - choose one of two options:
+    - *Manufacturer* - select this option, if you have a gateway from a manufacturer with a Gateway EUI. The following field is available for this option:
+        - *Endpoint name - Gateway EUI*: A 64-bit extended unique identifier for your gateway. It should be provided by the manufacturer or printed on the gateway packaging. It will become the endpoint name by which the platform server will identify the device.
+    - *No gateway EUI* - select this option if your gateway doesn't have a Gateway EUI. The following field is available for this option:
+        - *Endpoint name*: Provide an endpoint name by which the platform server will identify your device.
+- *Gateway ID*:  Unique identifier for your gateway that will be used by The Thinks Stack to identify your device.
+- *Friendly name - Gateway name*: An optional human-readable name to help you identify your gateway. The Things Stack will use this name instead of the gateway ID to refer to your gateway.
+
+In the section *Settings*, select your Frequency plan. The frequency plan defines data rates that your end device or gateway is set up to use. The gateway and end devices within reach must use the same frequency plan to communicate. 
+
+In the section *LwM2M security mode for device-server interaction* select the security mode:
+
+- Pre-shared key - authorize your device using a pre-shared key. To use this option, provide the following:
+    - Key identity - define the name that the device will use during the DTLS handshake. Can be equal to the LwM2M device endpoint name.
+    - Key format. 
+    - Key - provide the shared secret used in the device-server authentication.
+- NoSec - use the device with no security established for the device-server communication. Use this mode only for testing purposes.
+- Certificate - secure your device-server communication with a certificate:
+    - Select *Use a previously uploaded certificate* if you have already uploaded a certificate to the platform using the DTLS/TLS Certificates panel.
+    - Select *Upload a new certificate* to load your certificate by clicking **Browse**.
```

Comment:
In the `In the section Device credentials, provide the following information:` section above those names (e.g. `Pre-shared key`) were in italics.